src/navigation/Tabs.js

```diff
@@ -13,51 +13,35 @@ import FontAwesome5 from 'react-native-vector-icons/FontAwesome5'
 
 const Tab = createBottomTabNavigator();
 
+import {iconConfigs} from "./iconsConfigs";
 export default function Tabs() {
-
     return (
         <NavigationContainer>
             <Tab.Navigator
-                screenOptions={({route})=>({
-                    tabBarIcon:({focused,size,color})=>{
-                        let iconName;
-                        if (route.name === 'Home'){
-                            iconName='home';
-                            size=focused? 25:20;
-                            color=focused? 'red':'#000';
-                        }else if (route.name === 'All books'){
-                            iconName='book';
-                            size=focused? 25:20;
-                            color=focused? 'red':'#000'
-                        }else if (route.name === 'List Of Books'){
-                            iconName='bookmark';
-                            size=focused? 25:20;
-                            color=focused? 'red':'#000'
-                        }else if (route.name === 'Book Shop'){
-                            iconName='shopping-basket';
-                            size=focused? 25:20;
-                            color=focused? 'red':'#000'
-                        }
-                        return(
-                            <FontAwesome5
-                                name={iconName}
-                                size={size}
-                                color={color}
-                            />
+                screenOptions={({route}) => ({
+                    headerStyle: {backgroundColor: '#000'},
+                    headerTintColor: 'red',
+                    headerTitleAlign: 'center',
+                    tabBarIcon: ({focused, size, color}) => {
+                        const props = iconConfigs.getConfig(route.name, focused)
+                        return (
+                            <FontAwesome5 name={props.iconName} size={props.size} color={props.color}/>
```

Comment:
Do you really need two kinds of icons (FeatherIcons and FontAwesome5) in your project? Or it's same? Please re-consider this moment and give yourself reasoned answer , because lots of different styles/icons/image styles in one project is always bad practice.